src/contentscript/contentscript.ts

```diff
@@ -221,7 +229,7 @@ let sendButtonKey = (b: number) => {
             break
         case 5:
             // Hangup
-            // (no keyboard shortcut)
+            // (no keyboard shortcut) <= there is one in fact with h
```

Comment:
Thanks for pointing this out.  I don't think it was there originally, or wasn't documented.

Reply:
Yes, this is possible, this is always evolving